var/spack/repos/builtin/packages/py-hieroglyph/package.py

```diff
@@ -14,9 +14,12 @@ class PyHieroglyph(PythonPackage):
     homepage = "https://github.com/nyergler/hieroglyph"
     pypi = "hieroglyph/hieroglyph-1.0.0.tar.gz"
 
+    version('2.1.0', sha256='b4b5db13a9d387438e610c2ca1d81386ccd206944d9a9dd273f21874486cddaf')
     version('1.0.0', sha256='8e137f0b1cd60c47b870011089790d3c8ddb74fcf409a75ddf2c7f2516ff337c')
     version('master')
```

Comment:
I think `master` should be removed, or a `git` attribute should be added. Right now it tries to download from PyPI and this version doesn't exist.

Reply:
version `master`removed.